.cirrus.yml

```diff
@@ -26,6 +26,7 @@ test_task:
     - RUSTFLAGS="$RUSTFLAGS -C target-feature=+lse" RUSTDOCFLAGS="$RUSTDOCFLAGS -C target-feature=+lse" cargo test -vv --workspace --exclude asm-test --exclude bench --all-features --release -- -Z unstable-options --report-time
     # Use -Z build-std because the prebuilt libtest seems to be incompatible with LTO, causing miscompilation: https://gist.github.com/taiki-e/9713f8e02e8f9f852ccee8d6f089ec24
     - RUSTFLAGS="$RUSTFLAGS -C target-feature=+lse" RUSTDOCFLAGS="$RUSTDOCFLAGS -C target-feature=+lse" CARGO_PROFILE_RELEASE_CODEGEN_UNITS=1 CARGO_PROFILE_RELEASE_LTO=fat cargo -Z build-std test -vv --workspace --exclude asm-test --exclude bench --all-features --release --tests --target $TARGET -- -Z unstable-options --report-time
+    # TODO: lse2 is not available on Graviton2 (armv8.2-a)
```

Comment:
When Cirrus CI moved to Graviton3 (armv8.5-a? IIRC) we can test this with real aarsh64 machines on CI.